### PR TITLE
fix apache healthcheck

### PR DESCRIPTION
**Pull request by Zoey2936**
Signed-off-by: Zoey zoey@z0ey.de

### CODE REVIEW

File: Containers/apache/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+if [ "$APACHE_PORT" != '443' ]; then
+    curl -skfI localhost:8000 || exit 1
+    curl -skfI localhost:$APACHE_PORT || exit 1
+else
```
Comment (Zoey2936) [inline Containers/apache/healthcheck.sh:4]:
```suggestion
    curl -skfI localhost:$APACHE_PORT || exit 1
    curl -skfI $NC_DOMAIN:443 || exit 1
else
```
I am thinking if this should be also added

Comment (szaimen) [reply Containers/apache/healthcheck.sh]: this will not work in all cases so I wouldn't do this...

File: Containers/apache/Dockerfile [hunk 2]
```diff
@@ -61,6 +61,7 @@ RUN mkdir /var/log/supervisord; \
 COPY Caddyfile /
 
 COPY start.sh /usr/bin/
+COPY healthcheck.sh /usr/bin/
 COPY supervisord.conf /
 RUN chmod +x /usr/bin/start.sh; \
```
Comment (szaimen) [inline Containers/apache/Dockerfile:66]:
```suggestion
RUN chmod +x /usr/bin/start.sh; \
    chmod +x /usr/bin/ healthcheck.sh; \
```

File: Containers/apache/healthcheck.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$APACHE_PORT" != '443' ]; then
+    curl -skfI localhost:8000 || exit 1
+    curl -skfI localhost:"$APACHE_PORT" || exit 1
+else
+    curl -skfI localhost:8000 || exit 1
```
Comment (szaimen) [inline Containers/apache/healthcheck.sh:7]:
```suggestion
curl -skfI localhost:8000 || exit 1
if [ "$APACHE_PORT" != '443' ]; then
    curl -skfI localhost:"$APACHE_PORT" || exit 1
else
    c
```

File: Containers/apache/healthcheck.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    curl -skfI "$NC_DOMAIN":"$APACHE_PORT" || exit 1
```
Comment (szaimen) [inline Containers/apache/healthcheck.sh:8]:
```suggestion
    curl -skfI https://"$NC_DOMAIN":"$APACHE_PORT" || exit 1
```

**Comment by szaimen** [comment]
Please squash. Then I'll merge!

**Comment by Zoey2936** [comment]
done

**Review by szaimen — APPROVED** [review]
Thanks!

**Pull request merged**